src/sentry/integrations/slack/unfurl/metric_alerts.py

```diff
@@ -67,28 +73,59 @@ def unfurl_metric_alerts(
         )
     }
 
-    return {
-        link.url: SlackMetricAlertMessageBuilder(
-            alert_rule=alert_rule_map[link.args["alert_rule_id"]],
-            incident=incident_map.get(link.args["incident_id"]),
+    orgs_by_slug: Dict[str, Organization] = {org.slug: org for org in all_integration_orgs}
+
+    result = {}
+    for link in links:
+        if link.args["alert_rule_id"] not in alert_rule_map:
+            continue
+        org = orgs_by_slug.get(link.args["org_slug"])
+        if org is None:
+            continue
+
+        alert_rule = alert_rule_map[link.args["alert_rule_id"]]
+        selected_incident = incident_map.get(link.args["incident_id"])
+
+        chart_url = None
+        if features.has("organizations:metric-alert-chartcuterie", org):
+            try:
+                chart_url = build_metric_alert_chart(
+                    organization=org,
+                    alert_rule=alert_rule,
+                    selected_incident=selected_incident,
+                    period=link.args["period"],
+                    start=link.args["start"],
+                    end=link.args["end"],
+                    user=user,
+                )
+            except Exception as e:
+                sentry_sdk.capture_exception(e)
```

Comment:
We typically use `logging.exception("Failed to build metric alert chart")`, although either is fine.